src/planscape/collaboration/permissions.py

```diff
@@ -49,6 +49,38 @@ def can_add_scenario(user: User, planning_area: PlanningArea):
         return check_for_permission(user.id, planning_area, "add_scenario")
 
 
+class PlanningAreaNotePermission(CheckPermissionMixin):
+    @staticmethod
+    def can_view(user: User, planning_area_note: PlanningAreaNote):
+        if is_creator(user, planning_area_note.planning_area):
+            return True
+        return check_for_permission(
+            user.id, planning_area_note.planning_area, "view_planningarea"
+        )
+
+    @staticmethod
+    def can_add(user: User, planning_area_note: PlanningAreaNote):
+        if is_creator(user, planning_area_note.planning_area):
+            return True
+        return check_for_permission(
+            user.id, planning_area_note.planning_area, "view_planningarea"
+        )
+
+    @staticmethod
+    def can_change(user: User, planning_area_note: PlanningAreaNote):
+        ## TODO nobody will be changing notes?
+        return is_creator(user, planning_area_note.planning_area) or is_creator(
```

Comment:
Forgot about this TODO, but we could either just return false here or follow the can_remove pattern below.